src/olympia/reviewers/views.py

```diff
@@ -831,6 +842,9 @@ def review(request, addon, channel=None):
         redirect_url = reverse('reviewers.unlisted_queue_all')
 
     if request.method == 'POST' and form.is_valid():
+        if comments_draft:
+            comments_draft.delete()
```

Comment:
I'd move this below `form.helper.process()` in case there's an exception thrown in there.

Reply:
Fixed in 2047d45